pkg/server/commands/listusers/list_users_rpc.go

```diff
@@ -178,22 +177,23 @@ func (l *listUsersQuery) ListUsers(
 	}()
 
 	go func() {
-		defer close(foundUsersCh)
 		internalRequest := fromListUsersRequest(req, &datastoreQueryCount)
-		if err := l.expand(cancellableCtx, internalRequest, foundUsersCh); err != nil {
-			expandErrCh <- err
-			return
-		}
+		err := l.expand(cancellableCtx, internalRequest, foundUsersCh)
+		close(foundUsersCh)
+		expandErrCh <- err
```

Comment:
I don't like the idea of sending `nil`s through a channel. That can cause a panic if the receiver forgets to check for nil

Reply:
Changed this so that we only send `if err != nil`